lib/internal/test_runner/utils.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+const { RegExpPrototypeExec } = primordials;
+const { basename } = require('path');
+const kSupportedFileExtensions = /\.[cm]?js$/;
```

Comment:
what about `.node`?

Reply:
Are there any projects running their test suites out of `.node` files that you can point me to? I don't think that is a pattern we should support and encourage. If you explicitly specify a `.node` file, it will work though (same with `.json` files). From what I can tell, that's how most other test runners work with `.node` files.

Reply:
nope, just seemed like a good thing to include for consistency.

If it explicitly works when you pass the path, then that's great! It seems worth calling this out in docs tho.